### DISCUSSION TITLE
How can I edit the mapping in Elasticsearch?

### DISCUSSION THREAD

**Discussion by YenseanHub** [OrchardCMS/OrchardCore#14902] [Q&A]
I want to add a filed to a property in elasticsearch mapping, But I dont know how 
For example,  I add a Media Field in a content type
and I want to ues the "analyzer : ik_max_word"
Now the mapping is just for reference only , So how can I add that filed to the mapping ?
thanks 
![screenshoot](https://github.com/OrchardCMS/OrchardCore/assets/29721016/b4ab56ca-db07-4dbd-987e-06e8b5e81b8f)

**Comment by Skrypt** [comment]
Refer to Elasticsearch documentation for doing that. It needs to be executed directly on the Elasticsearch server Kibana dashboard or you will require to know how to use Nest https://github.com/elastic/elasticsearch-net
